protocol/serialization-types/tests/test_deserialize_snark_work.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
```

Comment:
Now the development is complete I suggest we delete this file use the standard method of `block_path_test_batch` in test-serialization.rs to cover the snark work data structure

Reply:
removed this file

Reply:
removed this file